### PR TITLE
Create InEtn56274.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: StPetersburg/InEtn56274.xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                    <locus from="2r" to="12r"/>
+                     <title type="complete" ref="LIT5842MedicalAdvice"></title>
+                     <textLang mainLang="am"/>
+                     <colophon xml:id="coloph1" xml:lang="am">
+                       <locus from="2v" to="3r"></locus>
+                       ከመስኮብ፡ ቋንቋ፡ በሀበሻ፡ ቋንቋ፡ የተረጐመው፡ <persName ref="PRS12879Bazzabbeh" role="translator">በዛብህ፡</persName> ነው።
+                       <date calendar="ethiopian" when-custom="04-14">በታኅሣሥ፡ በ፲፬ ቀን፡</date>
+                       <placeName ref="LOC0001IHA">ባዲስ፡ አበባ፡</placeName> ከተማ፡ ተጻፈ። የደረሰውም፡ ሐኪም፡ <persName ref="PRS8661scusev" role="author">ጴጥሮስ፡ ሹሲፍ፡</persName>
```
Comment (thea-m) [inline StPetersburg/InEtn56274.xml]: I think it's better to use the BM record LOC1216AddisA

File: StPetersburg/InEtn56274.xml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <textClass>
+            <keywords>
+               <term key="AmharicLiterature"/>
+               <term key="Medicine"></term>
```
Comment (thea-m) [inline StPetersburg/InEtn56274.xml]: and translation?

File: StPetersburg/InEtn56274.xml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                    <locus from="2r" to="12r"/>
+                     <title type="complete" ref="LIT5842MedicalAdvice"></title>
+                     <textLang mainLang="am"/>
+                     <colophon xml:id="coloph1" xml:lang="am">
+                       <locus from="2v" to="3r"></locus>
+                       ከመስኮብ፡ ቋንቋ፡ በሀበሻ፡ ቋንቋ፡ የተረጐመው፡ <persName ref="PRS12879Bazzabbeh" role="translator">በዛብህ፡</persName> ነው።
+                       <date calendar="ethiopian" when-custom="04-14">በታኅሣሥ፡ በ፲፬ ቀን፡</date>
```
Comment (thea-m) [inline StPetersburg/InEtn56274.xml]: I am not sure how this date should be marked up. The calendar "ethiopian" refers to the ʿĀmata śəggāwe, and here there is no reference to the year, only the month and day, so I think this value should better not be used. There would be the possibility for the mark up with ethiocal:Tahsas14, which has been used until now for recurring dates for liturgical readings etc (not a precise, but incomplete date as here).

File: StPetersburg/InEtn56274.xml (hunk 1 above)
Comment (DariaElagina) [reply StPetersburg/InEtn56274.xml]: done

File: StPetersburg/InEtn56274.xml (hunk 2 above)
Comment (DariaElagina) [reply StPetersburg/InEtn56274.xml]: done

File: StPetersburg/InEtn56274.xml (hunk 3 above)
Comment (DariaElagina) [reply StPetersburg/InEtn56274.xml]: corrected, i used the format provided by TEI for such incomplete dates.

**Comment by DariaElagina** [comment]
can it be merged?

**Comment by thea-m** [comment]
Yes, sorry, I was still wondering about the "ethiopian" in the date. It is probably fine.

**Pull request merged**